src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
   // extract the json from the response
   const result = response.data.choices[0].text ?? ''
+
+  // eslint-disable-next-line no-console
```

Comment:
### Line 27
## CodeGuard Suggestions
**Suggestion:** Consider using a more secure form of logging, such as a log monitoring solution, to debug code behavior and issues.
**Reason:** To make the code more secure